cmd/get_logs.go

```diff
@@ -255,25 +272,46 @@ func (glc *getLogsCmd) collectLogs(node v1.Node, config *ssh.ClientConfig) (stri
 }
 
 func (glc *getLogsCmd) uploadScript(node v1.Node, client *ssh.Client) (string, error) {
-	if isWindowsNode(node) || glc.linuxScriptPath == "" {
+	if glc.windowsScriptPath == "" && glc.linuxScriptPath == "" {
```

Comment:
I would rather reduce code duplication like it was before

Reply:
updated

Reply:
I am not quite sure what you mean here. Do you mean we should remove this if condition?

Reply:
Updated, thanks for the clarification